File: task-03/Elixir/SubTask_2.ex
case File.read("input.txt") do
  {:ok,content} ->
    case File.write("output.txt",content) do
      :ok ->
        IO.puts "copied to output.txt"
      {:error,reason} ->
        IO.puts "#{reason}"
    end
  {:error,reason} ->
    IO.puts "#{reason}"
end

File: task-03/Rust/SubTask_2.rs
use std::fs::File;
use std::io::{self,Read,Write};

fn main()->io::Result<()> {
    let mut input=File::open("input.txt")?;
    let mut data=String::new();
    input.read_to_string(&mut data)?;
    let mut output=File::create("output.txt")?;
    output.write_all(data.as_bytes())?;
    Ok(())
}
